src/main/java/com/dnd/runus/client/vo/AppleAuthTokenResponse.java

```diff
@@ -0,0 +1,13 @@
+package com.dnd.runus.client.vo;
+
+
+public record AppleAuthTokenResponse(
+    String access_token,
+    String expires_in,
+    String id_token,
+    String refresh_token,
+    String token_type,
+    String error
+) {
```

Comment:
Java 코드에는 Camel case를 사용하면 좋을 것 같아요!

`@JsonNaming`을 클래스 상단에 적거나,
`@JsonProperty("access_token") String accessToken`와 같은 방식으로 적을 수 있어요!

```suggestion
@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
public record AppleAuthTokenResponse(
    String accessToken,
    String expiresIn,
    String idToken,
    String refreshToken,
    String tokenType,
    String error
) {
```